app/components/ButtonGroup/ButtonGroup.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function ButtonGroup(props) {
+  const { buttons, } = props;
+  const buttonHeight = {
```

Comment:
This is not "buttonHeight". It is "style".

Reply:
done